cmd/tk/util.go

```diff
@@ -56,3 +58,22 @@ func writeJSON(i interface{}, path string) error {
 
 	return nil
 }
+
+// writeJsonnet writes the given object to the path as a formatted Jsonnet file
+func writeJsonnet(i interface{}, path string) error {
+	out, err := json.MarshalIndent(i, "", "  ")
+	if err != nil {
+		return fmt.Errorf("marshalling: %s", err)
+	}
+
+	main, err := formatter.Format(path, string(out), formatter.DefaultOptions())
```

Comment:
This should use `tanka.Format` defined in https://github.com/grafana/tanka/blob/17baa356eaea5a30c4a88a8950e2dbefb51636e1/pkg/tanka/format.go#L88-L90

It uses the same functionality underneath, but if we ever might modify the way we format things, we only need to look in one place

Reply:
I noticed, Javier also made a comment on another PR related to writing and formatting files. I hope to consolidate that in another PR.